course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -181,9 +181,64 @@ describe('Unit testing PeopleController', function() {
 
     });
     describe('confirmAddPeopleToCourse', function() {
-        it('reflects the right role and number of people');
-        it('initiates the process if confirmed');
-        it('doesn\'t initiate the process if canceled');
+        beforeEach(function() {
```

Comment:
There's now a [one-liner](https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/blob/task/elliottyates/tlt-2119/course_info_add_multiple_people_jasmine_tests/course_info/tests/jasmine/controllers/PeopleControllerSpec.js#L39) that might be able to replace this CI mocking stuff:
```
clearInitialCourseInstanceFetch();
```